vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -113,17 +111,16 @@ func NewClusterStateFeeder(config *rest.Config, clusterState *model.ClusterState
 	factory := informers.NewSharedInformerFactory(kubeClient, defaultResyncPeriod)
 	controllerFetcher := controllerfetcher.NewControllerFetcher(config, kubeClient, factory)
 	return ClusterStateFeederFactory{
-		PodLister:             podLister,
-		OOMObserver:           oomObserver,
-		KubeClient:            kubeClient,
-		MetricsClient:         newMetricsClient(config),
-		VpaCheckpointClient:   vpa_clientset.NewForConfigOrDie(config).AutoscalingV1beta2(),
-		VpaLister:             vpa_api_util.NewAllVpasLister(vpa_clientset.NewForConfigOrDie(config), make(chan struct{})),
-		ClusterState:          clusterState,
-		LegacySelectorFetcher: target.NewBeta1TargetSelectorFetcher(config),
```

Comment:
I can see that LegacySelectorFetcher is deleted, so it means that v1beta1 objects will be read as v1beta2 and marked as UnsupportedConfig since they don't have target ref. Is it expected? Don't we want to have additional information for the user that they use v1beta1?

Reply:
Added more info to the error. Thanks@